README.md

```diff
@@ -6,129 +6,207 @@
+# Market Maker Keeper Setup Guide 
+
+**Guide Outline**
+1. Introduction 
+2. Prerequisites 
+3. Installation 
+4. Testing 
+5. Bands and Bands Configuration
+    - Example
+6. Order Rate Limitation
+7. Data Templating Language
+8. Price Feed Configuration
+    - Example 
+9. Running Keepers
+    - Example (OasisDEX) 
+10. Known limitations
+11. Support Information
+12. License
+
+## 1. Introduction
+A big part of the DAI Stablecoin System (DSS) is the incentivization of external agents, called **Keepers** (which can be human but are typically automated bots). Market Maker Keepers work by creating a series of orders in so-called **bands** (defined later), which are configured with a JSON file containing parameters like spreads, maximum engagement, etc. In short, the `market-maker-keeper` repository is a set of Keepers that facilitate market making on exchanges. For example, trading Dai motivated by the expected long-term convergence toward the indicated `Target Price`. This guide is dedicated to showing you how to create your very own Market Maker Keeper Bot as well as educate the community and help both users and developers understand the value of this incredible software. We are proud to say that all of the code needed to get a Market Maker Keeper bot up and running is open-sourced.
+
+### List of current exchanges that Market Maker Keeper Bots can be built for
+
+- OasisDEX (`oasis-market-maker-keeper`)
+- EtherDelta (`etherdelta-market-maker-keeper`)
+- RadarRelay and ERCdEX (`0x-market-maker-keeper`)
+- Paradex (`paradex-market-maker-keeper`)
+- DDEX (`ddex-market-maker-keeper`)
+- Ethfinex (`ethfinex-market-maker-keeper`)
+- GoPax (`gopax-market-maker-keeper`)
+- OKEX (`okex-market-maker-keeper`)
+- TheOcean (`theocean-market-maker-keeper`)
+
+## 2. Prerequisite
+- Git
+- [Python v3.6.6](https://www.python.org/downloads/release/python-366/)
+- [virtualenv](https://virtualenv.pypa.io/en/latest/)
+    - This project requires *virtualenv* to be installed if you want to use Maker's python tools. This helps with making sure that you are running the right version of python and checks that all of the pip packages that are installed in the **install.sh** are in the right place and have the right versions.
+- [X-code](https://apps.apple.com/ca/app/xcode/id497799835?mt=12) (for Macs)
+
+## 3. Getting Started (Installation)
+
+1. **Clone the `market-maker-keeper` repository and switch into its directory:**
+```
+git clone git@github.com:makerdao/market-maker-keeper.git
+cd market-maker-keeper 
+```
+2. **Initializing the git submodules that will bring in both the pymaker and the pyexchange library.** 
 ```
-git clone https://github.com/makerdao/market-maker-keeper.git
-cd market-maker-keeper
 git submodule update --init --recursive
-./install.sh
-```
-
-For some known Ubuntu and macOS issues see the [pymaker](https://github.com/makerdao/pymaker) README.
+   ``` 
 
-## Testing
+3. **Set up the virtual env and activate it:**
```

Comment:
Next steps (setting up virtual env, activate, install prereq) are automatically done by install.sh which should be easier than manually do that, is there any reason you removed it?